src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -444,7 +444,7 @@ protected static uint ComputeArgHash(TexlNode[] args)
 
         public virtual bool SupportCoercionForArg(int argIndex)
         {
-            return SupportsParamCoercion && (argIndex <= MinArity || argIndex <= MaxArity);
+            return SupportsParamCoercion && (argIndex >= MinArity || argIndex <= MaxArity);
```

Comment:
>>= [](http://example.com/codeflow?start=54&length=2)

not related to change, but it has to be a bug.

Reply:
Can you do this in a separate PR?  Especially since this could impact PA and have wide impact

Reply:
removed